### ISSUE TITLE
Compatibility with sbt 1.0

### ISSUE THREAD

**Issue by callicles** [scoverage/sbt-coveralls#95]
Sbt was released in 1.0 version, a release for that version would be nice.

Thank you

**Comment by ruippeixotog** [comment]
I also need a SBT 1.0 version for sbt-coveralls, it's the last missing plugin in several of my projects. Is #96 enough for supporting 1.0?

**Comment by jozic** [comment]
@ruippeixotog it should be enough, I published it locally and it works fine on my project which uses sbt 1.0.x
would love to see #96 merged and released

**Comment by jozic** [comment]
Can be closed now, 1.2.2 is released yesterday

**Comment by MasseGuillaume** [comment]
The README should be updated as well: 

http://repo1.maven.org/maven2/org/scoverage/sbt-coveralls_2.12_1.0/1.2.2/
http://repo1.maven.org/maven2/org/scoverage/sbt-scoverage_2.12_1.0/1.5.1/

```scala
addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.3.5")
addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.1.0")
```

to

```
addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
addSbtPlugin("org.scoverage" % "sbt-coveralls" % "1.2.2")
```

**Comment by jozic** [comment]
@MasseGuillaume see #100

**Comment by gslowikowski** [comment]
Closed by #100
